lib/validateSchema.js

```diff
@@ -30,20 +32,20 @@ function validateSchema(schema, options) {
 }
 
 function validateObject(schema, options) {
-	var validate = ajv.compile(schema);
-	var valid = validate(options);
+	const validate = ajv.compile(schema);
+	const valid = validate(options);
 	return valid ? [] : filterErrors(validate.errors);
 }
 
 function filterErrors(errors) {
-	var newErrors = [];
-	errors.forEach(function(err) {
-		var dataPath = err.dataPath;
-		var children = [];
-		newErrors = newErrors.filter(function(oldError) {
-			if(oldError.dataPath.indexOf(dataPath) >= 0) {
+	let newErrors = [];
+	errors.forEach((err) => {
+		const dataPath = err.dataPath;
+		let children = [];
```

Comment:
It was recommended in an earlier comment to re-assign `children` on line 48 so I think this has to stay a `let`. thanks!